runner/model/service.go

```diff
@@ -88,6 +88,7 @@ const (
 
 // Affinity describes the affinity rules of a service
 type Affinity struct {
+	ID               string             `json:"id" yaml:"id"`
```

Comment:
What is this for?

Reply:
@HeyItsJs said he wanted something to uniquely identify an affinity object

Reply:
Shouldn't you store it somewhere then? Would become a problem while getting the affinities in k8s

Reply:
I am not storing it anywhere, while fetching the service, i'll generate a new kid